test/e2e/specs/fakerp/etcdbackuprecovery.go

```diff
@@ -28,7 +28,9 @@ var _ = Describe("Etcd Recovery E2E tests [EtcdBackupRecovery][Fake][LongRunning
 	)
 
 	BeforeEach(func() {
-		backup, err := random.LowerCaseAlphanumericString(5)
+		var err error
+		// := can't be used below, because that redeclares "backup" in local scope
```

Comment:
We could solve this by not mutating `backup = foo + backup` below maybe and calling it something else (like `backup_label`? Is that the bug?

Reply:
This will work too, and is cleaner :)
The bug is that there's var backup string declared in line 26, and using := declares a new backup string in the scope of BeforeEach func, so none of the init code has any effect on the value of "backup" in the rest of the code.